docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -296,6 +296,26 @@ quarkus.rest-client.extensions-api.verify-host=false
 This setting should not be used in production as it will disable the SSL hostname verification.
 ====
 
+=== HTTP/2 Support
+
+By default, HTTP/2 is disabled in REST Client. If you want to enable it, you can set:
+
+[source, properties]
+----
+quarkus.rest-client.http2=true
```

Comment:
Hm, maybe it should be enable-http2? @gsmet WDYT?

Reply:
For my understanding, it doesn't make sense to enable HTTP/2 always and let the client negotiate with the server?

Reply:
Actually, in vert.x http core this is enabled by default, so it's fine with me to enable http2 by default. Please, confirm if you want me to do this change here. 

About, being just `http2` or `enable-http2`, I like more the latter, but in HTTP configuration, it's simply `http2`. Anyway, let me know what you prefer and I will change it. The same for the alpn property.

Reply:
You mean in our http configuration for the server? If so, then let's be consistent

Reply:
Yes, so will keep the current 'https' and 'alpn' names

Reply:
I assume you mean `http2` 🙂

Reply:
🤦

Reply:
> Actually, in vert.x http core this is enabled by default, so it's fine with me to enable http2 by default.

Let's use the default Vert.x uses

Reply:
PR updated enabling http2 by default.

Reply:
I think we probably need @cescoffier 's blessing to enable it by default.

Reply:
From my point of view and taking into account the number of failures that enabling http2 by default in REST Client has caused, this flag should be disabled by default. 

Though I'm trying to find a solution to the Multipart failures, the solution seems to not be trivial as it involves Vert.x client + Netty client (at least, not trivial to me! Since I don't have much experience with either Vert.x or Netty).

Reply:
One thing I don't understand is why this now fails if http2 is enabled by default in Vert.x HTTP client

Reply:
I think this is only a client thing that is caused by the QuarkusMultipartFormUpload implementation which internally uses Netty HTTP 1/1. But I'm still investigating.